### PR TITLE
Fix stat and conditionals to evaluate it is the current scene

### PR DESCRIPTION
**Pull request by BoredX**

### CODE REVIEW

File: src/client/Client.java [hunk 1]
```diff
@@ -106,6 +117,11 @@ private void updateFrame(int choiceNum) {
                     currScene = scene;
                 }
             }
+            if (oldScene.equals(currScene.name)) {
```
Comment (mrbrendanwong) [inline src/client/Client.java]: Can you give some insight as to when this occurs?

File: src/client/Client.java [hunk 2]
```diff
@@ -81,13 +83,22 @@ public void launchFrame(){
 
     // Updates current elements of game on the frame
     private void updateFrame(int choiceNum) {
+        // Current scene
+        String oldScene = currScene.name;
```
Comment (mrbrendanwong) [inline src/client/Client.java]: Would prefer this named prevScene just for clarity

File: src/client/Client.java (hunk 1 above)
Comment (thethomasz) [reply src/client/Client.java]: if the scene that was written in the code doesnt actually exist

Comment (mrbrendanwong) [reply src/client/Client.java]: Ahh alright. Thanks

**Comment by mrbrendanwong** [comment]
@BoredX feel free to merge. Looks good

**Pull request merged**